lib/cylc/cfgspec/suite.py

```diff
@@ -315,6 +318,16 @@ def _coerce_final_cycletime( value, keys, args ):
                 'execution timeout'           : vdr( vtype='interval_minutes'),
                 'reset timer'                 : vdr( vtype='boolean', default=False ),
                 },
+            'events' : {
+                'handler'                     : vdr( vtype='string_list', default=[] ),
```

Comment:
This should really be "handlers" plural.  I think it is OK for the following two items to be singular though, as they  apply to each handler individually.